product/dialogs/miq_dialogs/miq_provision_redhat_dialogs_clone_to_template.yaml

```diff
@@ -0,0 +1,496 @@
+---
```

Comment:
seems like this doesn't belong to this PR?

Reply:
How did that get there.... 

I'll remove it.

Reply:
Removed.